src/main/java/edu/cornell/kfs/pmw/batch/service/PaymentWorksWebServiceConstants.java

```diff
@@ -22,12 +24,22 @@
         public static final String PAYMENTWORKS_AUTHORIZATION_TOKEN = "paymentworks.authorization.token";
     }
 
+    public static final class PaymentWorksCommonJsonConstants {
+        public static final String DETAIL_FIELD = "detail";
+        public static final String STATUS_FIELD = "status";
+        public static final String STATUS_OK = "ok";
+    }
+
     public static final class PaymentWorksTokenRefreshConstants {
```

Comment:
No, the refresh-token feature has not been regression tested with this yet. Once the revisions are ready, I'll work on spinning up TEMP so that I can test that endpoint again.